src/components/Map.js

```diff
@@ -11,7 +11,7 @@ function Map(){
 
     };
     return (
-        <div style={{ height: '100vh', width: '100%' }}>
+        <div style={{ height: 'calc(66.67vh - 1.25rem)', width: '100%' }}>
```

Comment:
You should be able to use `100%` for the height because the container is set to `calc(66.67vh - 1.25rem)`. Please try that so we can retain control of the height of the map via CSS. Thanks!